main.tf

```diff
@@ -28,7 +28,8 @@ resource "aws_instance" "app_server" {
   ebs_block_device {
     device_name = "/dev/sda1"
     volume_size = var.DISK_SIZE_GB
-    volume_type = "gp2"
```

Comment:
We mention gp2 disk in the README. Can you update that?
Thanks

Reply:
[3f00ad4](https://github.com/maticnetwork/matic-cli/pull/96/commits/3f00ad4a42802fb9c3a5e179754ca418eca85878)